### PR TITLE
Spelling

### PR DESCRIPTION
**Pull request by jsoref**
Thank you for contributing to the CF CLI! Please read the following:

* Please make sure you have implemented changes in line with the [contributing guidelines](https://github.com/cloudfoundry/cli/blob/master/.github/CONTRIBUTING.md)
* We're not allowed to accept any PRs without a signed CLA, no matter how small.
If your contribution falls under a company CLA but your membership is not public, expect delays while we confirm.
* All new code requires tests to protect against regressions.
* Contributions must be made against the appropriate branch. See the [contributing guidelines](https://github.com/cloudfoundry/cli/blob/master/.github/CONTRIBUTING.md)
* Contributions must conform to our [style guide](https://github.com/cloudfoundry/cli/wiki/CLI-Product-Specific-Style-Guide). Please reach out to us if you have questions.

## Does this PR modify CLI v6, CLI v7, or CLI v8?

Not meaningfully afaict

## Description of the Change

This PR corrects misspellings identified by the [check-spelling action](https://github.com/marketplace/actions/check-spelling).

The misspellings have been reported at https://github.com/jsoref/cloudfoundry-cli/commit/b700f2163153a9ac80525141d48919cbfb8c4b0f#commitcomment-87631508

The action reports that the changes in this PR would make it happy: https://github.com/jsoref/cloudfoundry-cli/commit/dfec2652cfa3c22fb63c9a3eaf6524f6f4c46e89
 
Note: this PR does not include the action. If you're interested in running a spell check on every PR and push, that can be offered separately.

## Why Is This PR Valuable?

Typos make products harder to use and code harder to debug

## Why Should This Be In Core?

Explain why this functionality should be in the cf CLI, as opposed to a plugin. 

## Applicable Issues

List any applicable Github Issues here

## How Urgent Is The Change?

This will be the least urgent PR you'll see this year or next. Although the longer it takes to process the more merge conflicts will accumulate.

## Other Relevant Parties

Who else is affected by the change?

### CODE REVIEW

File: cf/commands/application/ssh_test.go [hunk 1]
```diff
@@ -281,7 +281,7 @@ var _ = Describe("SSH command", func() {
 
 			Context("Error when connecting", func() {
 				It("notifies users", func() {
-					fakeSecureShell.ConnectReturns(errors.New("dial errorrr"))
+					fakeSecureShell.ConnectReturns(errors.New("dial error"))
```
Comment (jsoref) [inline cf/commands/application/ssh_test.go:284]: This repository is one of the cuter repositories I've seen in a while. For things which were clearly slang, I've left them alone. For things that were borderline, I sometimes changed them. This is borderline.

File: cf/flags/flags_test.go [hunk 2]
```diff
@@ -73,7 +73,7 @@ var _ = Describe("Flags", func() {
 				Ω(fCtx.Bool("skip2")).To(Equal(true), "skip2 should be true")
 				Ω(fCtx.Bool("name")).To(Equal(false), "name should be false")
 				Ω(fCtx.String("name")).To(Equal("johndoe"), "name should be johndoe")
-				Expect(fCtx.Bool("non-exisit-flag")).To(Equal(false))
+				Expect(fCtx.Bool("non-exist-flag")).To(Equal(false))
```
Comment (jsoref) [inline cf/flags/flags_test.go:76]: I'd normally want to change this to `nonexistent`, but I'm not sure if `non` is magical here.

File: cf/terminal/table_test.go [hunk 3]
```diff
@@ -254,13 +254,13 @@ var _ = Describe("Table", func() {
 
 		It("supports multi-byte French runes", func() {
 			table = NewTable([]string{"", "", "", "", "", ""})
-			table.Add("nom", "état demandé", "instances", "mémoire", "disque", "adresses URL")
+			table.Add("nom", "état demandé", "instances", "mémoire", "disque", "addresses URL")
```
Comment (jsoref) [inline cf/terminal/table_test.go:257]: Oops, this is a French run.

File: util/ui/ui_for_push_test.go [hunk 4]
```diff
@@ -35,7 +35,7 @@ var _ = Describe("UI", func() {
 	})
 
 	Describe("DisplayChangesForPush", func() {
-		It("alings all the string types", func() {
+		It("aligns all the string types", func() {
```
Comment (jsoref) [inline util/ui/ui_for_push_test.go:38]: ??

**Review by jsoref — COMMENTED** [review]
Many corrections suggested by Google Sheets. All fault mine (and the biggest mistakes were manual)

**Review by a-b — APPROVED** [review]
Great effort! LGTM, thanks.

The only question is how can we port this to V7 and V8?

**Comment by a-b** [comment]
@jsoref can you please prepare PRs for v8 and v7 based on [f773611](https://github.com/cloudfoundry/cli/commit/f773611a17174b5a10dbb7810b9f3187ab436540)

**Comment by jsoref** [comment]
@a-b done

**Pull request merged**